adflow/pyADflow.py

```diff
@@ -5631,6 +5881,42 @@ def _readPlot3DSurfFile(self, fileName, convertToTris=True):
 
         return pts, conn
 
+    # mham
+    def mhamDOT_slipvelocitiesfinelevel_block_d(self, useoldcoor, t, sps):
+        """ helper function to make dot product test"""
```

Comment:
This is never used in pyADflow but was most likely used during development to get the differentiated vals for slip velocities. The comments look good, but is it something we want to keep here or move to the docs?

Reply:
I would say this belongs in the docs